src/renderer/components/dashboard/Send.vue

```diff
@@ -160,11 +160,43 @@ export default {
       }
     },
 
-    send() {
+    async send() {
+      let fee;
+
+      if (this.currency.symbol === 'BTC') {
+        this.sending = true;
+
+        this.$services.neo.sendFundsBTC(this.address, this.amount)
```

Comment:
There is one condition in backend function, it will check that the user has sufficient funds or not, then it will go for the transaction.
https://github.com/aphtoken/aph-wallet-gui/blob/bde951e5d7d8c1b4dc8290f25244bfbb9876a5c1/src/renderer/services/neo.js#L1024-L1026